src/status_im/chat/events.cljs

```diff
@@ -301,14 +278,13 @@
 
 (handlers/register-handler-fx
   :navigate-to-chat
-  [(re-frame/inject-cofx :get-stored-messages) re-frame/trim-v]
+  [re-frame/trim-v]
```

Comment:
It would be great to decide if we want to generalize `trim-v`. If not, let's drop it.

Reply:
agree, let's drop `trim-v` and `register-handler-db` because it's better to use one approach in the whole code, but not in this PR

Reply:
Yes, in order to do that consistently, it would be a *lot* of additional changes in this PR. Let's discuss it and do it (and only that) consistently in some another PR.